### PR TITLE
Feat entities bus busroutes buscompanies

### PR DESCRIPTION
**Pull request by RedonShabani**
Feat entities bus busroutes buscompanies.

Created Bus, BusRoutes and BusCompanies entities and added them as tables in db.

### CODE REVIEW

**Review by ganilala — DISMISSED** [review]

**Review by edonhoxha04 — DISMISSED** [review]

**Review by oltahasimja — DISMISSED** [review]

File: pentasharp/Models/Entities/BusCompany.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.Models.Entities
+{
+    public class BusCompany
+    {
+        [Key]
```
Comment (GentaSL) [inline pentasharp/Models/Entities/BusCompany.cs:13]: add xml summaries https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

File: pentasharp/Models/Entities/BusRoutes.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.Models.Entities
+{
+    public class BusRoutes
+    {
+        [Key]
```
Comment (GentaSL) [inline pentasharp/Models/Entities/BusRoutes.cs:13]: add xml summaries https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

File: pentasharp/Models/Entities/Buses.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
+﻿using pentasharp.Models.Enums;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace pentasharp.Models.Entities
+{
+    public class Buses
+    {
+        [Key]
```
Comment (GentaSL) [inline pentasharp/Models/Entities/Buses.cs:15]: add xml summaries https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags

**Review by GentaSL — CHANGES_REQUESTED** [review]
add missing summaries as requested

**Review by ganilala — APPROVED** [review]

**Review by GentaSL — APPROVED** [review]

**Pull request merged**